packages/html/ds/src/footer/footer.html

```diff
@@ -0,0 +1,57 @@
+{% macro govieFooter(props) %}
+
+{% from 'heading/heading.html' import govieHeading %}
+{% from 'footer/helpers.html' import hasTwoCols,hasTwoColsLinks, govieFooterLink %}
+
+<footer class="gi-bg-gold-50 gi-p-x-xl gi-py-3xl gi-border-solid gi-border-t-xs gi-border-gold-500 gi-pt-3xl gi-pb-2xl" data-module="gieds-footer">
+  {# TODO: Use Container component #}
```

Comment:
question: Will this comment trimmed down and not come under copy-code block on storybook?